src/Libraries/Nop.Core/Domain/Messages/NewsLetterSubscription.cs

```diff
@@ -19,6 +19,8 @@ public partial class NewsLetterSubscription : BaseEntity
     /// Gets or sets a value indicating whether subscription is active
     /// </summary>
     public bool Active { get; set; }
+    public int SubscriptionLanguage { get; set; }

```

Comment:
Better to use the LanguageId as a name. And also we need a doc string to the new field.